CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [1.2.8](https://github.com/ably/ably-flutter/tree/v1.2.8)
+
+**Bug Fix:**
+push activation not completing on iOS when running in release configuration [\#276](https://github.com/ably/ably-flutter/issues/276),
```

Comment:
Might want to put a capital letter at the start here?

Reply:
It doesn't look to bad [when rendered](https://github.com/ably/ably-flutter/blob/release/1.2.8/CHANGELOG.md), but you're right it would look better with a capital. I'll fix that. Thanks.

Reply:
https://github.com/ably/ably-flutter/pull/279/commits/8ae7438918d15a0235257bc1180bac8727849693